success-criteria.md

```diff
@@ -417,6 +393,10 @@ stakeholder group.
             -   Access to adult content by an appropriate guardian to prevent
                 unauthorized viewing by underage family members
 
+            -   Tracking that would record any behavioral or user data of those under the age of 13
+
```

Comment:
The preceding point does not, as I understand it, cover even preexisting regulations in the body of COPPA. We should be sure that case is covered in our concerns.